<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgReforma</class>
 <widget class="QDialog" name="dlgReforma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lugar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLugar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarLugar">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Orden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtOrden"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckMasculino">
       <property name="text">
        <string>Masculino</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btOtrosDatos">
       <property name="text">
        <string>&amp;Otros datos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
